www/js/control/ProfileSettings.tsx

```diff
@@ -434,6 +435,7 @@ const ProfileSettings = () => {
           desc={authSettings.opcode}
           descStyle={settingStyles.monoDesc}></SettingRow>
         <DemographicsSettingRow></DemographicsSettingRow>
+        <CustomLabelSettingRow />
```

Comment:
This is a good place to put the `<CustomLabelSettingRow>`, but when testing I noticed that `<CustomLabelSettingRow>` shows up in all configurations, even if there are no multilabel choices. Some programs don't have "mode" and "purpose" labels - they just have surveys. If that's the case, custom labels don't apply.

We should show this row conditionally, like below (or a different way if you prefer)
```suggestion
        {appConfig?.survey_info?.['trip-labels'] == 'MULTILABEL' && <CustomLabelSettingRow />}
```